SIT.Manager/ViewModels/Play/CharacterSummaryViewModel.cs

```diff
@@ -161,4 +169,23 @@ private async Task Play()
             await errorDialog.ShowAsync();
         }
     }
+
+    private async Task Logout()
+    {
+        AkiCharacter? character = _connectedServer.Characters.FirstOrDefault(x => x.Username == Profile.Username);
+
+        if (character != null)
+        {
+            _connectedServer.Characters.Remove(character);
+
+            int index = _configService.Config.BookmarkedServers.FindIndex(x => x.Address == _connectedServer.Address);
+
+            if (index != -1 && !_configService.Config.BookmarkedServers[index].Characters.Any(x => x.Username == character.Username))
+            {
+                _configService.Config.BookmarkedServers[index].Characters.Remove(character);
+            }
```

Comment:
I may also be wrong about this but since `_connectedServer` is passed to us as a reference, the characters list there should be the same character list that's referred to here

Reply:
I actually based this code on the code that already saved the login details (assuming one had ticked remember me) - a literal copy/paste (but with Remove rather than Add of course).  But I'll check this too and make any required changes.

Reply:
You are 100% bang on.  So I am changing the code to:

```
_connectedServer.Characters.Remove(character);
_configService.UpdateConfig(_configService.Config);
```

The same can also be done for the save login code (line 147 onwards), I've tested it and what is there can be removed and replaced with:

```
_connectedServer.Characters.Add(character);
_configService.UpdateConfig(_configService.Config);
```

Are you happy for this to be optimised too or would you rather it was left as is?

Reply:
Sure! I'll probably end up forgetting about it if it's left to me ;P

Reply:
Just pushed a commit to resolve this issue.